optuna/samplers/_grid.py

```diff
@@ -158,6 +168,10 @@ def sample_independent(
         param_distribution: BaseDistribution,
     ) -> Any:
 
+        if "grid_id" not in trial.system_attrs:
+            message = "You should specify all parameters in enqueue_trial when using GridSampler."
```

Comment:
```suggestion
            message = "All parameters must be specified when using GridSampler with enqueue_trial."
```
It's mandatory, not a suggestion. ;)